README.md

```diff
@@ -4,17 +4,21 @@ Sauce Labs SDK for the Test-Composer API.
 
 # Usage
 
-## Create Reports
-
+## Initial TestComposer Client
 ```javascript
-const {Region, TestComposer} = require('@saucelabs/testcomposer');
+const { TestComposer } = require('@saucelabs/testcomposer');
 
 const client = new TestComposer({
   region: 'us-west-1',
   username: process.env.SAUCE_USERNAME,
   accessKey: process.env.SAUCE_ACCESS_KEY,
   headers: {'User-Agent': `your-fancy-reporter/1.2.3`}
 });
+```
+
+## Create Reports for JavaScript Batch Frameworks
```

Comment:
Is referring to "Batch Frameworks" here helpful? "Batch" is a name we use internally and probably doesn't connote anything meaningful to most users.

Reply:
What about just `JavaScript Frameworks`?